### DISCUSSION TITLE
Green box widget for Android/IOS

### DISCUSSION THREAD

**Discussion by codad5** [community/community#80927] [Mobile]
It would be nice if android can have the GitHub activity green box widget, so that once can be able to add it to their holme screen .

An app [GitHub Widget](https://play.google.com/store/apps/details?id=jp.co.chlorocube.githubcontributions) perform this but it would br nice if in built 

Thanks

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @codad5! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
